spec/clojure_ttt/presenter_spec.clj

```diff
@@ -0,0 +1,22 @@
+(ns clojure-ttt.presenter-spec
+  (:require [speclj.core :refer :all]
+            [clojure-ttt.presenter :refer :all]))
+
+(describe "presenter"
+  (it "displays gamemode options"
+    (should-contain "Type 0 to play human vs human"
+      (with-out-str (show-gamemode-options))))
+
+  (it "displays the board correctly"
+    (should-contain "x--\nx--\n---"
+      (with-out-str (show-board ["x" "-" "-" "x" "-" "-" "-" "-" "-"]))))
```

Comment:
You should use keywords to represent marks, rather than strings.
Strings have individual areas of memory, whereas keywords share areas, like constants. I.e. "x" "x" takes up 2 unit of memory, whereas :x :x takes up just 1 unit.